File: labs/lab4/Python/main.py
from math import factorial

n = int(input('Enter an integer number n > 0, n = '))
x = int(input('Enter an integer number, x = '))
k: int = 0
for k in range(0, n):
    k = k + 1
    print(pow(x, k), "/", factorial(2 * k), "\n")

File: labs/lab4/C++/main.cpp
#include <iostream>

using namespace std;


long chislitel(int k, int x) {
    int i = 2;
    long m = x;
    while (i <= k) {
        m *= x;
        i++;
    }
    return m;
}

long long znamenatel(int k) {
    long long factorial = 1;
    for (int i = 1; i <= (2 * k); i++) {
        factorial = factorial * i;
    }
    return factorial;
}

int main() {
    int n;
    cout << "Enter an integer number n > 0, n = ";
    cin >> n;
    int x;
    cout << "Enter an integer number x, x = ";
    cin >> x;
    int k = 1;
    while (k <= n) {
        cout << chislitel(k, x) << "/" << znamenatel(k) << "\n";
        k++;
    }
    return 0;
}

